### PR TITLE
Feature: Button hints on HUD

### PR DESCRIPTION
**Pull request by LostTrackpad**
**What does this pull request add?**
This pull request adds a button hint feature to the HUD during gameplay to help players learn controls.

**Preview**
image
*(Action is climbing a ladder.)*

**Known problems**: Nothing aside from ***missing localization for languages beside English.***

### CODE REVIEW

File: beatrun/gamemodes/beatrun/gamemode/cl/ButtonHints.lua [hunk 1]
```diff
@@ -0,0 +1,185 @@
+-- This file has lots of strings that needs to be localized.
```
Comment (JonnyBro) [inline beatrun/gamemodes/beatrun/gamemode/cl/ButtonHints.lua:1]: I think you can remove this line

Comment (LostTrackpad) [reply beatrun/gamemodes/beatrun/gamemode/cl/ButtonHints.lua:1]: Oh yeah I forgot about that, this *does* support localization now.

**Comment by JonnyBro** [comment]
Added Russian translation for button hints, I'll check it later today (so words don't go past their designated places)

**Comment by LostTrackpad** [comment]
> (so words don't go past their designated places)
What do you mean by this? The text is already drawn with draw.DrawText with the option to right-align.

**Comment by JonnyBro** [comment]
> What do you mean by this? The text is already drawn with draw.DrawText with the option to right-align.

ah, just to be sure

**Comment by LostTrackpad** [comment]
Fair enough, GLua sometimes glitches out and just makes things much harder than it needs to be.

**Comment by JonnyBro** [comment]
So, are we merging this?

**Comment by LostTrackpad** [comment]
**ABORTING** - This Git history is a fucking disaster.

**Pull request closed without merging**